### DISCUSSION TITLE
Explore: Word highlighting is not working for fields enabled with eye-icon

### DISCUSSION THREAD

**Discussion by Theoooooo** [grafana/grafana#56248] [Observability]
Hello there,

I'm making this issue because this fonctionnality, who was recently added, is not working on my side.

## How to reproduce the issue

- Go to the explore tab
- Select a valid Elasticsearch datasource with data inside it
- Make a quick research with a term that i know for sure is present inside my indices
- Seeing that my term researched is not being highlighted

## Proof of the issue

![proof_its_not_working](https://user-images.githubusercontent.com/33425208/113148009-aab56700-9231-11eb-973b-14a6b6746f85.png)

## Additionnal Informations
Grafana Version : 7.5.2
Elasticsearch version : 7.12.0

Plus, the screenshot used to demonstrate this new functionnality is **very very** small. I can't even see if there is an actual manipulation to highlitgh my research.

I this functionnality is working only based on the "Message field name" option inside the data source, it's not making sense if your trying to find these terms inside message with multiples field inside them.

The option in question inside the datasource configuration :
![image](https://user-images.githubusercontent.com/33425208/113148226-e6e8c780-9231-11eb-931f-be719b65d4ec.png)

**Comment by Elfo404** [comment]
Hi @Theoooooo , thanks for opening this.

currently the highlight functionality works only when not using detected fields, so only for the configured message field name (which fallbacks to the first string property when not set).

I'm marking this as a feature request, it's not something we considered when we implemented it.

**Comment by Theoooooo** [comment]
Hello @Elfo404 

That's exactly what i though when i tried to see the replicate the exemple from the screenshot posted inside the news for the 7.5v version.

Glad to see this feature would be added for the other fields 👍

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
